tx-tools/src/sse.ts

```diff
@@ -35,6 +35,6 @@ app.post("/messages", async (req, res) => {
   await transport.handlePostMessage(req, res);
 });
 
-app.listen(3001, () =>
-  logger.info("Terminology Tools server listening on port 3001"),
+app.listen(3004, () =>
```

Comment:
let's leave the default port as it was - 3001.
And instead, let's add new env variable, PORT that will be used instead of 3001, and 3001 should be default value if it's not passed.

And further, in Docker images pass ENV=8000 for all images - it's required for uniformity of docker images.